### ISSUE TITLE
Results : 

### ISSUE THREAD

**Issue by aaaclones** [bszek213/ml_mlb#3]
After running the development branch, this is what I received. I'm unsure about how to interpret these results. Could you please explain how to interpret them?

<img width="1280" alt="Screenshot 2023-09-24 at 1 20 02 AM" src="https://github.com/bszek213/ml_mlb/assets/104782042/a7b1be73-42eb-4523-a242-2dc3c7d8f675">

**Comment by bszek213** [comment]
Yeah this is tricky part. As of right now I am doing no team-against-team comparisons, so I can't have "team a has a 3% change of betting team b." its mainly just if this teams plays like this their next game, the model thinks that it has an x% chance of betting whomever they face. I am going to change this in the future.

**Comment by aaaclones** [comment]
Wow , this was quite disappointing . But goodluck and thanks for the
clairity
